docusaurus/docs/learn/core-concepts/pki.md

```diff
@@ -1,89 +1,56 @@
-first and used to produce the certificates specified in the `cert` and `server_cert` fields of the Ziti Controller
-configuration file.
+The trust bundle defined in controller config property `identity.ca` is used by the controller to bundle known CAs' certificates. This bundle is downloaded by clients during enrollment from the client API in `/edge/client/v1/.well-known/est/cacerts` and subsequently used to verify controller and router server certificate chains. These known CAs are never used by the controller to verify client certificates which are always from the controller's own edge enrollment CA or a known, verified external CA.
 
-**cert**: The certificate presented to other Ziti Network components during connection establishment.
+#### Controller Configuration Reference
 
-**server_cert**: The certificate returned by the Ziti Controller when other Ziti Network components attempt to
-communicate to the Ziti Controller over the IP and port specified in the `ctrl.listener` or `mgmt.listener` fields of the Ziti Controller
-configuration file. If an edge section is present in the configuration file and no edge.api.identity section exists this
-certificate is also returned to incoming connections to the `edge.api.advertise` endpoint.
+Please refer to [the configuration reference](../../reference/30-configuration/conventions.md#identity) for a description of each property in the conventional `identity` section that is used in the controller's configuration file.
 
 ### Edge Router
 
 An Edge Router has one section related to PKI: `identity`. It is important to note that an Edge Router will
 manage its own PKI. Allowing the Edge Router to manage its own PKI is almost certainly desired. The
 only setting that an operator may wish to provide is the `key` field of the identity. This field is treated differently
 than the other values specified.  If the `key` specified does not exist a new key will be generated. If the `key`
-provided exists the Edge Router will use it and the other fields will be **regenerated and overwritten** as necessary.
-
-The certificate generated will be signed by the Ziti Controller using the certificate specified in `edge.enrollment.signingCert`.
-
-#### PKI Configuration
-
-The `identity` section of the Edge Router configuration is used by the Edge Router when connections are
-established to or from the other components of a Ziti Network. There are four sections in the identity block:
-`cert`, `server_cert`, `key`, `ca`.
-
-**ca**: A file representing a group of certificates with one or more certificate chains terminating at a trust anchor.
-When a Ziti Network component connects to the Edge Router and offers a certificate for validation the incoming
-connection is checked to see if it signed by a trust anchor specified in this file.
+provided exists the Edge Router will use it and the other fields will be **regenerated and overwritten** as necessary, if the filesystem is writable.

```

Comment:
What happens if the file system isn't writable?

Reply:
Nothing happens if it's not writable. Should we say that?